tests/PhpUnit/JsonSchemaConstraintTest.php

```diff
@@ -53,7 +54,8 @@ public function testValidSchema()
 JSON;
         $response = json_decode($response);
 
-        self::assertTrue($this->constraint->evaluate($response, '', true), $this->constraint->evaluate($response));
```

Comment:
The test is stateless, but because evaluate() is called twice, matches() and check() are also called twice